backend/drm/drm.c

```diff
@@ -461,6 +461,17 @@ static uint32_t get_possible_crtcs(int fd, uint32_t conn_id) {
 	return 0;
 }
 
+static void wlr_drm_connector_set_dpms(struct wlr_output *output, enum dpms_enum level) {
+	struct wlr_drm_connector *conn = (struct wlr_drm_connector *)output;
+	struct wlr_drm_backend *drm = (struct wlr_drm_backend *)output->backend;
+
+    if (drmModeObjectSetProperty (drm->fd, conn->id,
```

Comment:
This is using the legacy DRM interface. DPMS levels have been removed from the atomic interface IIRC. Should we handle DPMS levels at all?

Reply:
Ok, must admit I'm no drm expert, so I looked at weston and mutter for reference and they both had levels. But if we shouldn't use levels, would wlr_drm_connector_enable fit my usecase then?
In that case I can throw away some code...

Reply:
Yeah, the initial plan was to use `wlr_output_enable`. I'm not a DRM expert either though, so I'm not sure it's the right thing to do™. I know that Mutter doesn't support atomic modesetting and that Weston only recently gained support for it, so they might not be the best examples to look at wrt. the atomic interface.

Reply:
cc @ascent12

Reply:
`wlr_drm_connector_enable` is already the DPMS function.

Reply:
Just tested wlr_output_enable, but that seemed to turn off the screen so it didn't wake up again.

Reply:
Must have been some other glitch... tested again and it works! 
So, that means this pull request is void. And can be discarded...